docs/output.md

```diff
@@ -27,40 +27,41 @@ For further reading and documentation see the [FastQC help](http://www.bioinform
 
 **Output directory: `results/QC_shortreads/fastqc`**
 
-- `sample_fastqc.html`
-  - FastQC report, containing quality metrics for your untrimmed raw fastq files
+- `[sample]_[1/2]_fastqc.html`: FastQC report, containing quality metrics for your untrimmed raw fastq files
+- `[sample]_unmapped_[1/2]_fastqc.html`: FastQC report, containing quality metrics for filtered read files
 
 ### fastp
 
-[fastp](https://github.com/OpenGene/fastp) is a all-in-one fastq preprocessor for read/adapter trimming and quality control. It is used in this pipeline for trimming adapter sequences and discard low-quality reads.
+[fastp](https://github.com/OpenGene/fastp) is a all-in-one fastq preprocessor for read/adapter trimming and quality control. It is used in this pipeline for trimming adapter sequences and discard low-quality reads. Output is in the results folder and part of the MultiQC report.
 
-**Output directory: `results/QC_shortreads/fastp`**
+**Output directory: `results/QC_shortreads/fastp/[sample]`**
 
-- The trimmed reads are not included in the output by default, but the `--keep_trimmed` will store the trimmed reads in this directory
+- `fastp.html`: Interactive report
+- `fastp.json`: Report in json format
 
-### remove_phix
+### Remove PhiX sequences from short reads
 
 The pipeline uses bowtie2 to map the reads against PhiX and removes mapped reads.
 
 **Output directory: `results/QC_shortreads/remove_phix`**
 
-- Contains a brief log file indicating how many reads have been removed.
+- `[sample]_remove_phix_log.txt`: Contains a brief log file indicating how many reads have been retained.
 
 ### Host read removal
 
 The pipeline uses bowtie2 to map short reads against the host reference genome specified with `--host_genome` or `--host_fasta` and removes mapped reads.
 
 **Output directory: `results/QC_shortreads/remove_host`**
 
-- Contains the bowtie2 log file indicating how many reads have been mapped as well as a file listing the read ids of discarded reads.
+- `[sample].bowtie2.log`: Contains the bowtie2 log file indicating how many reads have been mapped as well as a file listing the read ids of discarded reads.
```

Comment:
```suggestion
- `[sample].bowtie2.log`: Contains the bowtie2 log file indicating how many reads have been mapped as well as a file listing the read ids of discarded reads.
```

The information about discarded and kept reads is also included in the MultiQC report

Reply:
Added this.